tests/datasets/event-json.ts

```diff
@@ -0,0 +1,11 @@
+export type EventJson = {
```

Comment:
A couple of things:

1) Type definitions should be inside *.d.ts files, as a general convention
2) This type alias is not used in other script files so I'd just put this at the top of the only test file where it's used, if really needed

Reply:
Done 👍🏻